node-commons/src/main/java/org/apache/iotdb/commons/schema/tree/AbstractTreeVisitor.java

```diff
@@ -296,10 +336,32 @@ protected final N getParentOfNextMatchedNode() {
     }
   }
 
+  /**
+   * Get level from root to current node. Level of root is 0. For example, root.sg.d1.s1,
+   * currentNode is s1, then return 3.
+   *
+   * @return level from root to current node
+   */
+  protected final int getLevelOfNextMatchedNode() {
```

Comment:
Anyway, the interface name and description and the parameter name provided for subclasses shall keep consistence, no matter ```NextMatchedNode``` or ```CurrentNode```.